File: src/ssr_pkg/src/yao_node.cpp
#include <ros/ros.h>
#include <iostream>
#include <std_msgs/String.h>

int main(int argc, char **argv)
{
    ros::init(argc, argv, "yao_node");
    std::cout << "辅助已经上号" << std::endl;
    ros::NodeHandle nh;
    ros::Publisher pub = nh.advertise<std_msgs::String>("help_me_get_more_score", 10);

    ros::Rate loop_rate(10.0);
    std_msgs::String msg;
    while (ros::ok())
    {
        msg.data = "带我飞我陪你睡";
        pub.publish(msg);
        loop_rate.sleep();
    }

    return 0;
}

File: src/ssr_pkg/scripts/yao_node.py
#! /usr/bin/env python3
#coding=utf-8

import rospy
from std_msgs.msg import String

if __name__=="__main__":
    rospy.init_node("yao_node")
    rospy.logwarn("辅助上线")
    pub=rospy.Publisher("help_me_get_more_score",String,queue_size=10)
    rate=rospy.Rate(10)
    msg=String()
    while not rospy.is_shutdown():
        msg.data="带我飞我陪你睡"
        pub.publish(msg)
        rate.sleep()